### PR TITLE
feat(extension): [LW-9028] Add wallet concepts to menu

### PR DESCRIPTION
**Pull request by renanvalentin**
# Checklist

- [x] JIRA - [LW-9028](https://input-output.atlassian.net/browse/LW-9028)
- [ ] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

Requires the PR https://github.com/input-output-hk/lace/pull/728 to be merged first. It will render the new UI when `process.env.USE_MULTI_WALLET === 'true'`

## Screenshots

Screenshot 2023-11-21 at 09 21 39
Screenshot 2023-11-21 at 09 21 51

[LW-9028]: https://input-output.atlassian.net/browse/LW-9028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/browser-extension-wallet/src/components/MainMenu/DropdownMenuOverlay/components/AddNewWalletLink.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+// import { useAnalyticsContext } from '@providers';
+// import {
+//   MatomoEventActions,
+//   MatomoEventCategories,
+//   AnalyticsEventNames,
+//   PostHogAction
+// } from '@providers/AnalyticsProvider/analyticsTracker';
+
+const handleOnClicked = () => {
+  // analytics.sendEventToMatomo({
```
Comment (lucas-barros) [inline apps/browser-extension-wallet/src/components/MainMenu/DropdownMenuOverlay/components/AddNewWalletLink.tsx:16]: Should this be removed?

Comment (renanvalentin) [reply apps/browser-extension-wallet/src/components/MainMenu/DropdownMenuOverlay/components/AddNewWalletLink.tsx:16]: I also added to the menu: 
Screenshot 2023-11-21 at 13 38 37

**Review by lucas-barros — APPROVED** [review]

**Comment by DominikGuzei** [comment]
@renanvalentin I can see the "copy adress" popout when hovering the wallet in the menu
image

**Comment by DominikGuzei** [comment]
@renanvalentin just saw this while working on my PR
image

this might have been introduced with the larger menu button?

**Comment by renanvalentin** [comment]
> @renanvalentin just saw this while working on my PR image
> 
> this might have been introduced with the larger menu button?

@DominikGuzei i couldn't reproduce it. maybe it was a previous build? 

Screenshot 2023-11-27 at 11 01 42

**Comment by DominikGuzei** [comment]
@renanvalentin yes I can confirm that I also cannot reproduce it anymore 👍

**Comment by DominikGuzei** [comment]
> > @renanvalentin just saw this while working on my PR image
> > this might have been introduced with the larger menu button?
> 
> @DominikGuzei i couldn't reproduce it. maybe it was a previous build?
> 
> Screenshot 2023-11-27 at 11 01 42

@renanvalentin yes I can confirm that I also cannot reproduce it anymore 👍

**Pull request merged**